heron/common/src/cpp/network/connection.cpp

```diff
@@ -114,13 +114,13 @@ sp_int32 Connection::registerForBackPressure(VCallback<Connection*> cbStarter,
 sp_int32 Connection::writeIntoIOVector(sp_int32 maxWrite, sp_int32* toWrite) {
   sp_uint32 bytesLeft = maxWrite;
   sp_int32 simulWrites =
-      mIOVectorSize > mNumOutstandingPackets ? mNumOutstandingPackets : mIOVectorSize;
+      mIOVectorSize > mOutstandingPackets.size() ?: mIOVectorSize;
```

Comment:
Is this standard c++? I thought it is GNU C++ extension. If this is part of the regular c++ standard, it is fine?